test/data/weatherSensors.mjs

```diff
@@ -0,0 +1,118 @@
+export const weatherSensors = {
+    hotStormy: function() {
```

Comment:
Nice collection of sensors for different weather patterns. You could even generate them dynamically. Look up "property based tests"